src/update_agent/actor.rs

```diff
@@ -464,6 +463,16 @@ impl UpdateAgentInfo {
         depls
     }
 
+    /// Return whether the update stream of the pending deployment, and cleanup the pending
```

Comment:
I'm failing to parse the "whether" here.

Reply:
ah, that was a typo. There shouldn't be a "whether" there.